modules/nextflow/src/main/groovy/nextflow/script/ProcessConfig.groovy

```diff
@@ -813,6 +813,13 @@ class ProcessConfig implements Map<String,Object>, Cloneable {
      *      The {@link ProcessConfig} instance itself.
      */
     ProcessConfig errorStrategy( strategy ) {
+        listOfStrategies = ['terminate', 'finish', 'ignore', 'retry']
+        if (strategy !instanceof Closure) {
+            if (!listOfStrategies.contains(strategy)) {
+                throw new IllegalArgumentException("Unrecognized error strategy: " + strategy + ". Available strategies are: " + listOfStrategies.join(', '))
```

Comment:
You can use string interpolation instead of `+` concatenation to make it more readable